src/reducers/VaccineReducer.js

```diff
@@ -32,6 +32,7 @@ export const VaccineReducer = (state = initialState(), action) => {
     case VACCINE_ACTIONS.SCAN_STOP: {
       return {
         ...state,
+        scannedSensorAddresses: [],
```

Comment:
resetting this when stopping a scan also - had a bit of jank when navigating back to the tab